pyogrio/_io.pyx

```diff
@@ -1523,23 +1523,26 @@ def ogr_read_info(
         fields = get_fields(ogr_layer, encoding)
 
         meta = {
-            'crs': get_crs(ogr_layer),
-            'encoding': encoding,
-            'fields': fields[:,2], # return only names
-            'dtypes': fields[:,3],
-            'geometry_type': get_geometry_type(ogr_layer),
-            'features': get_feature_count(ogr_layer, force_feature_count),
-            'total_bounds': get_total_bounds(ogr_layer, force_total_bounds),
-            'driver': get_driver(ogr_dataset),
+            "name": get_string(OGR_L_GetName(ogr_layer)),
+            "crs": get_crs(ogr_layer),
+            "encoding": encoding,
+            "fields": fields[:,2], # return only names
+            "dtypes": fields[:,3],
+            "fid_column": get_string(OGR_L_GetFIDColumn(ogr_layer)),
```

Comment:
```suggestion
            "fid_field": get_string(OGR_L_GetFIDColumn(ogr_layer)),
```
(for consistency, since we refer to the list of columns as "fields")

Reply:
> since we refer to the list of columns as "fields"

Yes, but on the other hand the geometry and FID columns/fields are not(?) part of the list of (attribute) fields in the `"fields"` entry.

Reply:
Yes, because I feel there are arguments possible in all directions, I just followed the GDAL naming, but I don't have explicit preferences...

Reply:
Ok - let's stick to what you have.